tdrs-backend/tdpservice/users/authentication.py

```diff
@@ -12,12 +13,22 @@ class CustomAuthentication(BaseAuthentication):
     def authenticate(username=None, login_gov_uuid=None, hhs_id=None):
         """Authenticate user with the request and username."""
         User = get_user_model()
-
+        logging.debug("CustomAuthentication::authenticate:hhs_id {}".format(hhs_id))
+        logging.debug("CustomAuthentication::authenticate:login_gov_uuid {}".format(login_gov_uuid))
+        logging.debug("CustomAuthentication::authenticate:username {}".format(username))
         try:
-            if login_gov_uuid:
-                return User.objects.get(login_gov_uuid=login_gov_uuid)
-            elif hhs_id:
```

Comment:
I needed to upend this order for users with pre-existing login_gov_uuid to update hhs_id else that path will fail and will bubble up a non-existent user to login.py's handle_user which attempts to create a user via e-mail at this line:
```
            user = User.objects.create_user(email, email=email, **auth_options)
```

This fails as the user w/ that email already exists so this user update must happen within `authenticate()`.